src/Cli/dotnet/commands/dotnet-test/Terminal/AnsiCodes.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
For all files that are copied from MTP, could we have a file comment or a special parent folder to identify them? This will go out of sync too quickly otherwise.